packages/core/src/utils/formatRuneName.test.ts

```diff
@@ -0,0 +1,47 @@
+import { describe, expect, it } from "vitest";
+import { formatRuneName } from "./formatRuneName";
+
+describe("core | utils | formatRuneName", () => {
+	it("return error if name is empty", () => {
+		expect(() => formatRuneName(" ")).toThrow("Rune name cannot be empty");
+	});
+
+	it("return error if rune name contains !", () => {
```

Comment:
We won't be able to validate all the possible characters existing in the unicode, test is redundant 

Ther eis no need to test each character in the separate test case. We just can try some chars: a few symbols, non-ascii chars etc.